.github/workflows/ci-rust.yml

```
@@ -146,13 +147,30 @@ jobs:
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
         timeout-minutes: 1
 
+      - name: Install winfsp
+        if: startsWith(matrix.os, 'windows')
+        shell: bash
+        run: |
+          set -eux
+          choco install winfsp -y --version=${{ env.WINFSP_VERSION }}
+          mkdir -vp ${{ runner.temp }}
+          curl -L https://github.com/winfsp/winfsp/releases/download/v2.0/winfsp-tests-${{ env.WINFSP_VERSION }}.zip -o ${{ runner.temp }}/winfsp-tests.zip
+          unzip ${{ runner.temp }}/winfsp-tests.zip -d ${{ runner.temp }}/winfsp-tests.exe
+
       # Install cargo nextest command
       - uses: taiki-e/install-action@7412a5eb82f4686390763b0f92637284e91b6627 # pin v2.15.2
         with:
           tool: nextest@${{ env.cargo-nextest-version }}
 
       # Don't run platform agnostic tests here (only runned on Linux)
 
+      - name: Test with winfsp-tests
+        if: startsWith(matrix.os, 'windows')
+        shell: bash
+        run: cargo nextest run ${{ env.CARGO_NEXTEST_CI_FLAGS }} --package winfsp_tests
+        env:
+          WINFSP_TEST_EXE: ${{ runner.temp }}/winfsp-tests.exe
```

According to the comment about the `unzip -d ..` that value need to be changed

```suggestion
          WINFSP_TEST_EXE: ${{ runner.temp }}/winfsp-tests/winfsp-tests.exe
```

---

Ty